frontend/pages/DashboardPage/cards/ActivityFeed/components/ActivityAutomationDetailsModal/ActivityAutomationDetailsModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import Modal from "components/Modal";
+import Button from "components/buttons/Button";
+import Textarea from "components/Textarea";
+import { IActivityDetails } from "interfaces/activity";
+
+const baseClass = "activity-automation-details-modal";
+
+interface IActivityAutomationDetailsModalProps {
+  details: IActivityDetails;
+  onCancel: () => void;
+}
+
+const ActivityAutomationDetailsModal = ({
+  details,
+  onCancel,
+}: IActivityAutomationDetailsModalProps) => {
+  const renderContent = () => {
+    return (
+      <>
+        <div className={`${baseClass}__modal-content`}>
+          <p>
+            Fleet was configured to send a JSON payload to this URL whenever a
```

Comment:
@noahtalerman Note that this is slightly different than the Figma copy (see my comment there). Fell free to suggest a reversion to what's in Figma if that makes more sense, and let me know if we should show currently configured URL (if any) rather than the URL as of when the enable/edit happened (guessing the latter is more correct since this is supposed to be an audit log).